00_introduction/21_architecture_root_server.txt

```diff
@@ -170,13 +163,12 @@ Node. It also contains an init script:
 
 ----
 
-The +rudder-agent+ package depends on a few common libraries and utilities:
+The +rudder-agent+ package depends on a few libraries and utilities:
 
 * +OpenSSL+
 * +libpcre+
-* +libdb+ (4.6 on Debian)
-* +uuidgen+ (utility from +uuid-runtime package+ on Debian)
-
+* +liblmdb+
```

Comment:
Actually, we bundle this on many platforms, don't we? So we should say that here, like:

liblmdb (on platforms where it is available as a package - on others the rudder-agent package bundles it)